advisor-ui/src/components/ConstraintPopover/ConstraintPopover.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import Select from "@mui/material/Select";
+
+export default function ConstraintPopover({ addConstraint, handleClose }) {
+  const [type, setType] = useState("");
+  const [value, setValue] = useState("");
+
+  const handleChangeType = (event) => {
+    setType(event.target.value);
+  };
+
+  const handleChangeValue = (event) => {
+    setValue(event.target.value);
+  };
+
+  const handleConstraintSubmit = (type, value) => {
+    if (type === "" || value === "") {
+      alert("Enter both a type and value!");
```

Comment:
This is fine for a first-pass, but I think it's a pretty rough UX. If you have time in the future I'd come back and update the [inputs](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation) to actually handle the validation if possible.